### ISSUE TITLE
Make suffix of generated query response types customizable

### ISSUE THREAD

**Issue by jlkiri** [dotansimha/graphql-code-generator#2110]
**Is your feature request related to a problem? Please describe.**

`componentSuffix` option of `typescript-react-apollo` plugin allows us to generate React Apollo components with arbitrary suffixes. The default is `XXXComponent` but many would like to generate something like `XXXQuery`. However, this format is already reserved for generated query response types, e.g.

```javascript
export type PlayerQuery = ({ __typename?: 'Query' } &
{ player: ({ __typename?: 'PlayerModel' } & Pick<PlayerModel, 'id' | 'username' | 'character'>) });
```

**Describe the solution you'd like**

Although type names and component names should not clash, it would be preferrable to be able to also customize names of generated query responses via some similar option in config. I think `querySuffix` is appropriate.

**Describe alternatives you've considered**

Is this better done somehow by extending the execution flow?

**Additional context**

**Comment by jlkiri** [comment]
I have a working solution but not sure if it is the best way. I can send a PR.

**Comment by dotansimha** [comment]
@jlkiri sure, can you please share it here or submit a PR? I :)

**Comment by dotansimha** [comment]
Available in 1.4.0 🎉
